gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.2-all.zip
```

Comment:
Unfortunately, this should remain bin so the build system remains quick and does not have to download 200mb each time it runs.

You can add a wrapper task that downloads all if you did this for IDE compatibility.

Reply:
Fixed - yes it was from IDE bug reports :)